vendor/github.com/docker/go-metrics/go.mod

```diff
@@ -3,12 +3,3 @@ module github.com/docker/go-metrics
 go 1.11
 
 require github.com/prometheus/client_golang v1.1.0
-
-exclude (
```

Comment:
These exclusions should not be removed. They were removed when running `go mod vendor` I'm not sure how to go about keeping them aside from manually adding them back in and making sure any future PR which vendors also adds them back in :(

Reply:
Partial solution might be append a `go mod edit` command to [https://github.com/Azure/ARO-RP/blob/master/hack/update-go-module-dependencies.sh](https://github.com/Azure/ARO-RP/blob/master/hack/update-go-module-dependencies.sh).

Doesn't fix `go mod vendor` but would fix `make vendor`.

Reply:
I'll play around with this and see if I can get it to work, thank you for the suggestion. I was having issues with getting `make vendor` to run successfully so I'll likely have to figure that out first. Another idea (longer term) could be to add these exclusions upstream.

Reply:
go mod edit is for our go.mod file in the root directory (which already excludes protobuf versions). I manually added the exclusions back into the respective go.mod files being vendored. I think longer term we need to stop relying on locally vendored modules for our production binary and move to using a GOPROXY which we populate and access locally and from CI.